doc/source/user/select-openstack-services.rst

```diff
@@ -1,6 +1,6 @@
-====================================
-Select OpenStack services to install
-====================================
+=========================
+Select OpenStack Services
```

Comment:
You called it manage openstack service in the PR to 1.0.

Reply:
fixed